Firestore/core/src/firebase/firestore/core/snapshot_version.h

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2018 Google
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_CORE_SNAPSHOT_VERSION_H_
+#define FIRESTORE_CORE_SRC_FIREBASE_FIRESTORE_CORE_SNAPSHOT_VERSION_H_
+
+#include "Firestore/core/src/firebase/firestore/model/timestamp.h"
+
+namespace firebase {
+namespace firestore {
+namespace core {
+
+/**
+ * A version of a document in Firestore. This corresponds to the version
+ * timestamp, such as update_time or read_time.
+ */
+class SnapshotVersion {
+ public:
+  explicit SnapshotVersion(
+      const firebase::firestore::model::Timestamp& timestamp);
+
+  const firebase::firestore::model::Timestamp& timestamp() const {
+    return timestamp_;
+  }
+
+  /** Creates a new version that is smaller than all other versions. */
+  static const SnapshotVersion& NoVersion();
```

Comment:
`NoVersion` is the Objective-C name for this property. Android calls this `None` which is a better name.

In general for names of these kinds of things we want to end up closer to Android/JS than to the old Objective-C names.

Reply:
done